cucumber/api/features/support/logs_helpers.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Utility methods for Logs steps
+#
+require 'open3'
+
+module LogsHelpers
+  @@log_location = "/src/conjur-server/log/development.log"
+
+  def save_amount_of_log_lines(bookmark)
+    @bookmarks ||= Hash.new
+    @bookmarks[bookmark] = amount_of_log_lines
+  end
+
+  def occurences_in_log_filtered_from_bookmark(bookmark, msg)
+    raise "Bookmark #{bookmark} doesn't exists" unless @bookmarks[bookmark].present?
+    amount = amount_of_log_lines
```

Comment:
why do we need this?

Reply:
can't we just use `amount_of_log_lines`?

Reply:
oh i see that each call to `amount_of_log_lines` reads the log and counts number. maybe just change to `current_amount` but not critical